### ISSUE TITLE
No Idea how to Use Console Debugging in Brackets

### ISSUE THREAD

**Issue by Ashtheyogi** [adobe/brackets#13770]
Hey everyone. So I am learning front end dev and super new to using this text editor. Please help me out in laymans plain english please! I've looked up different solutions on how to use this but I just can't understand what they are saying in the troubleshooting. My issues:

1. In live preview, is it supposed to update immediately as you are writing code? Or do you have to save it every time to get the update?

**2.** In chrome, when I go to inspect my javascript in the console, my live preview turns off!! It's frustrating because it is slowing down my workflow alot

**3**. I tried using the debugging tools in the actual Brackets app and I have no idea what I am doing. I am working my save file of jspractice.html and when I look at the console in the brackets app debugging tool, it doesn't show me where my errors are. It shows errors from somewhere else,  and they are usually many lines down from what my project is. I have no idea what to do. I have a pic of the incorrect console stuff
![devtoolsconsole](https://user-images.githubusercontent.com/16600380/31579264-9d01a95e-b100-11e7-8ecf-c0d10a632b06.JPG)

Has anyone else figured out what this is all about? Thanks in advance!

**Comment by redmunds** [comment]
> 1. In live preview, is it supposed to update immediately as you are writing code? Or do you have to save it every time to get the update?

CSS and HTML updates are shown immediately, but you need to save JS to see updates.

> 2. In chrome, when I go to inspect my javascript in the console, my live preview turns off!! It's frustrating because it is slowing down my workflow alot

Correct. You cannot use chrome dev tools in live preview mode (because dev tools are required for live preview).

> 3. I tried using the debugging tools in the actual Brackets app and I have no idea what I am doing. I am working my save file of jspractice.html and when I look at the console in the brackets app debugging tool, it doesn't show me where my errors are.

Correct. The code you are seeing is for Brackets. This is for debugging Brackets itself.
